OracleDatabase/SingleInstance/dockerfiles/18.4.0/setPassword.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+ORACLE_PWD=$1
+
+su -p oracle -c "sqlplus / as sysdba << EOF
```

Comment:
This needs to be changed to reflect CDB architectures, i.e. also set the `SYS`,`SYSTEM`,PDBADMIN` passwords for the PDB.

Reply:
This file still needs to be changed to also set `SYS`,`SYSTEM`,`PDBADMIN` for the PDB.